packages/devtools_app/lib/src/screens/memory/framework/screen_body.dart

```diff
@@ -52,7 +63,7 @@ class _ConnectedMemoryBodyState extends State<ConnectedMemoryBody>
 
   @override
   Widget build(BuildContext context) {
-    if (!memoryController.isInitialized.value) {
+    if (!_initialized) {
```

Comment:
use a FutureBuilder instead so we don't need to add the boiler plate above. See other screens for examples.

Reply:
https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/profiler/profiler_screen.dart/#L122